### PR TITLE
Application Gateway for Multi Hosting template

### PR DESCRIPTION
**Pull request by puneetsaraswat**

### CODE REVIEW

File: 201-application-gateway-multihosting/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                            "password": "[parameters('certPassword1')]"
+                        }
+                    },
+                    {
+                        "name": "appGatewaySslCert2",
+                        "properties": {
+                            "data": "[parameters('certData2')]",
+                            "password": "[parameters('certPassword2')]"
+                        }
+                    },
```
Comment (bingosummer) [inline 201-application-gateway-multihosting/azuredeploy.json]: Need to remove `,`?

File: 201-application-gateway-multihosting/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,291 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "addressPrefix": {
```
Comment (singhkay) [inline 201-application-gateway-multihosting/azuredeploy.json]: Call this vnetAddressPrefix for clarity

**Comment by singhkay** [comment]
@puneetsaraswat Any update on this?

File: 201-application-gateway-multihosting/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Application Gateway for Multi Hosting",
+  "description": "This template creates an Application Gateway and configures it for Multi Hosting on port 443.",
+  "summary": "Application Gateway for Multi Hosting",
+  "githubUsername": "puneetsaraswat",
+  "dateUpdated": "2015-12-25"
```
Comment (singhkay) [inline 201-application-gateway-multihosting/metadata.json]: Needs an update

**Comment by singhkay** [comment]
@puneetsaraswat Make sure you update the dateUpdated in metadata file

**Comment by singhkay** [comment]
Skipping CI since this needs certs to be generated, merging.

**Pull request merged**